examples/custom-dialogs-demo/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This one's interesting. Do we want to port it, given that we now have a modal plugin?

Reply:
This is different to the modal plugin and I would actually not call it custom dialogs and more custom prompts. It's just overriding the built in prompts we use from browser alerts / confirms to dialogs. 
The modal plugin is more like a custom built in "modal UI" piece that can be used for different purposes (including possibly prompts.

I think this points at what we were talking about yesterday, the need to reassess our demos and see where they fit (potentially removing some / renaming some / adding additional descriptions / READMEs).

Reply:
For the purposes of this PR, I'll keep as is, however happy to remove from index.md if we want less eyes on it.